### ISSUE TITLE
Games only load in Windowed Mode

### ISSUE THREAD

**Issue by mralexs5** [NVIDIAGameWorks/rtx-remix#30]
**Describe the bug**
- All games that are using RTX Remix crash if not in windowed mode
- NVIDIA GeForce RTX 3060 Laptop GPU
- 528.2.0
- All

**To Reproduce**
1. Launch game in fullscreen
2. Crash

**Expected behavior**
I believe games should be able to run in fullscreen with this

**Comment by AlexDunn** [comment]
Thanks for this.

Are you still seeing this with the 0.1.1 hotfix that just went out recently?

0.1.0 definitely had this issue.

**Comment by mralexs5** [comment]
Yes, its still doing this on .1.1, at least for me

**Comment by yoyoinneverland** [comment]
> Yes, its still doing this on .1.1, at least for me, tested with the game Vampire: The Masquerade: Bloodlines

Did you manage to load into a game at all? I'm getting a failed to lock vertex buffer error
![image](https://user-images.githubusercontent.com/130618708/231633448-8da0725a-e5b7-46c9-b577-ce0e483ff09d.png)

**Comment by NV-DM** [comment]
Thank you for reporting. We're looking into VtMB. REMIX-1194 for tracking.

**Comment by mirrorcult** [comment]
loads in windowed mode for me, but as before HUD in game doesn't function

**Comment by Rmurray2006** [comment]
make a short cut on desktop and go to the shortcut patch area and down at the bottom, theres a drop down menu click maximized then go change your in game ini or whatever to window mode

**Comment by Rmurray2006** [comment]
there used to be a fake fullscreen app but it does not work sadly

**Comment by NV-DM** [comment]
Hello. Please retest on latest build.

**Comment by NV-LL** [comment]
Hello! Will you please check on the latest release and let us know if this issue persists? Thank you!

**Comment by CattoSalad** [comment]
Hello, I have tried it with the latest remix runtime and it's still an issue. 
I had to add the `-fullscreen` launch argument after launching the game with `-window` as a launch argument. I am guessing this is a quirk of the game.

I get the following error when launching the game in full screen:
![image](https://github.com/NVIDIAGameWorks/rtx-remix/assets/71943884/a710ef6a-18b5-45f5-9515-ca581ccf6cfa)

I did run the game without the runtime to ensure it did work in full screen before copying over the remix runtime files.

Hope this helps, thanks.

**Comment by NV-LL** [comment]
Thanks for confirming! What game(s) did you test on this time? And will you please send along any log files from that latest attempt to launch in fullscreen?

**Comment by nv-ajaus** [comment]
@CattoSalad @mralexs5 , I've made a change to dxvk-remix that I believe might help with this issue on Vampire the masquerade bloodlines. Can you please try using the latest dxvk-remix build (from here: https://github.com/NVIDIAGameWorks/dxvk-remix/actions)?

**Comment by CattoSalad** [comment]
I have tested it and it has fixed it. I had some random issues, but I don't believe these are related.
I had to add `-fullscreen`  after having run the game with the `-window` launch argument.
I did get some weird crash but I wasn't able to reproduce it. This occurred when I pressed new game and it just crashed mid loading.
It does also seem to stop responding sometimes when ALT+TABing out, but not a big issue in my opinion.

Thank you very much!
